packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -57,6 +60,18 @@ async function getTeamMembers({
     },
     distinct: ["userId"],
   });
+
+  const membershipWithUserProfile = [];
+  for (const membership of memberships) {
+    membershipWithUserProfile.push({
+      ...membership,
+      user: await UserRepository.enrichUserWithItsProfile({
+        user: membership.user,
+      }),
+    });
+  }
+
+  return membershipWithUserProfile;
```

Comment:
Why are we returning the profile with this endpoint? I tried looking where this is called and I don't think we need it.

Reply:
It is being used by buildMember(). I simply removed this code and that started erroring in TS